lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm_2019/ashrae_90_1_prm_2019.Model.rb

```diff
@@ -55,4 +55,264 @@ def model_get_percent_of_surface_range(model, wwr_parameter)
     end
     return wwr_range
   end
+
+  # Modify the existing service water heating loops to match the baseline required heating type.
+  # @param model [OpenStudio::Model::Model] OpenStudio model object
+  # @param building_type [String] the building type (For consistency with the standard class, not used in the method)
+  # @param swh_building_type [String] the swh building are type
+  # @return [Boolean] returns true if successful, false if not
+
+  def model_apply_baseline_swh_loops(model,
+                                     building_type,
+                                     swh_building_type = 'All others')
+    # Get the original water heater information
+    original_water_heater_info_hash = {}
+    model.getWaterHeaterMixeds.sort.each do |water_heater|
+      original_water_heater_info_hash = {water_heater.name.get.to_s => model_get_object_hash(water_heater)}
+    end
```

Comment:
Why do we need the original water heater info? Is it designed for the multi SHW workflow?

Reply:
Yes, it is for multi SHW. Maybe we could move it next to the multi SWH code.